Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/CompositeSequences/Validation/EconomizerMultiZone_Mod_DamLim.mo

```diff
@@ -0,0 +1,186 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.CompositeSequences.Validation;
+model EconomizerMultiZone_Mod_DamLim
+  "Validation model for multizone VAV AHU economizer operation: damper modulation and minimum ooutdoor air requirement damper position limits"
+  extends Modelica.Icons.Example;
+
+  parameter Real TOutCutoff(unit="K", quantity="TermodynamicTemperature")=297
+    "Outdoor temperature high limit cutoff";
+  parameter Real hOutCutoff(unit="J/kg", quantity="SpecificEnergy")=65100
+    "Outdoor air enthalpy high limit cutoff";
+  parameter Real TCooSet(unit="K", quantity="TermodynamicTemperature")=291
+    "Supply air temperature cooling setpoint";
+  parameter Real TSup(unit="K", quantity="TermodynamicTemperature")=290
+    "Measured supply air temperature";
+
+  parameter Modelica.SIunits.VolumeFlowRate minVOut_flowSet=0.71
+    "Example volumetric airflow setpoint, 15cfm/occupant, 100 occupants";
+  parameter Modelica.SIunits.VolumeFlowRate minVOut_flow=0.705
+    "Minimal measured volumetric airflow";
+  parameter Modelica.SIunits.VolumeFlowRate VOut_flowIncrease=0.03
+    "Maximum volumetric airflow increase during the example simulation";
+
+  parameter Types.FreezeProtectionStage freProDisabled = Types.FreezeProtectionStage.stage0
+    "Indicates that the freeze protection is disabled";
+  parameter Integer freProDisabledNum = Integer(freProDisabled)-1
+    "Numerical value for freeze protection stage 0 (=0)";
+  parameter Types.OperationMode occupied = Types.OperationMode.occupied
+    "AHU operation mode is \"Occupied\"";
+  parameter Integer occupiedNum = Integer(occupied)
+    "Numerical value for \"Occupied\" AHU operation mode";
+  parameter Types.ZoneState deadband = Types.ZoneState.deadband
+    "Zone state is deadband";
+  parameter Integer deadbandNum = Integer(deadband)
+    "Numerical value for deadband zone state (=2)";
```

Comment:
Don't use "(=2)". The point of enumerations is that one can change it for example by adding another element anywhere in the enumeration. If we end up changing it, we likely won't notice that somewhere in the code there is a comment that says deadband is 2, which will be wrong if a new element is added at the beginning of an enumeration.